src/Modifiers/CoreModifiers.php

```diff
@@ -2061,6 +2061,10 @@ public function replace($value, $params)
      */
     public function reverse($value)
     {
+        if (Compare::isQueryBuilder($value)) {
+            return $value->get()->reverse()->values()->all();
```

Comment:
You should be able to just get the collection from the query here, and then the reverse can happen in the next step.

```suggestion
            $value = $value->get();
```

Reply:
Whoops! Totally right, updated.

Reply:
An AssetCollection is a Collection. 👍

Reply:
Sorry, ignore me... :sweat_smile: